### PR TITLE
Fix nextPage null, Dual StartAnim in some case, Add navigateTo Feedback.

### PR DESCRIPTION
**Pull request by PascalPiche**
Fix nextPage bug when start anim is longer than loading the content.
Fix isAnimating detection not working well when two step anim, trigger two start anim.
Add boolean return value to navigateTo indicating if something append (new article animation triggered)

### CODE REVIEW

File: src/com/article-changer-v3.js [hunk 1]
```diff
@@ -121,73 +122,74 @@
 					
 					if (!nextPage.length) {
 						App.log({
-							args: 'Could not find new article',
+							args: 'Could not find new article with this handle: ' + (!!newPageHandle ? newPageHandle : '(empty handle)'), // jshint ignore:line
```
Comment (nitriques) [inline src/com/article-changer-v3.js]: `args` can be an array. so:

```suggestion
							args: [
								'Could not find new article with this handle: ',
								(!!newPageHandle ? newPageHandle : '(empty handle)')
							],
```

File: src/com/article-changer-v3.js [hunk 2]
```diff
@@ -82,12 +82,13 @@
...
 		var isAnimating = false;
 		var loadingUrl = '';
+		var nextPage;
 
 		var checkStartAnimAnd = function (current, next, o) {
 			isAnimating = false;
 			if (!isLoading) {
 				//Complete anim
-				o.endAnimToArticle(current, next, o);
+				o.endAnimToArticle(current, !!next && next.length ? next : nextPage, o);
```
Comment (nitriques) [inline src/com/article-changer-v3.js]: This looks scary to me. Really ? We can't avoid this ?

Comment (nitriques) [reply src/com/article-changer-v3.js]: @PascalPiche  ??

Comment (PascalPiche) [reply src/com/article-changer-v3.js]: Nope. scope problem. in some case next will be null.

**Pull request merged**